Re-enable eval script

rohansjoshi opened the pull request:
Summary:
Revised diff to enable eval_llama_qnn script which does evaluations with prefill model

Will add unit test in a later diff

Differential Revision: D82143474

cccclai reviewed (approved):
Thanks! Please add a unit test to guard this flow

rohansjoshi committed:
Re-enable eval script (#14176)

Summary:
Revised diff to enable eval_llama_qnn script which does evaluations with prefill model

Will add unit test in a later diff


Test Plan:
Imported from GitHub, without a `Test Plan:` line.

Rollback Plan:

Reviewed By: cccclai

Differential Revision: D82143474

Pulled By: rohansjoshi
